packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -225,6 +226,23 @@ mixin SchedulerBinding on BindingBase, ServicesBinding {
   AppLifecycleState get lifecycleState => _lifecycleState;
   AppLifecycleState _lifecycleState;
 
+  /// Initializes the [lifecycleState] with the initialLifecycleState from the
+  /// window.
+  ///
+  /// Once the [lifecycleState] is populated through any means (including this
+  /// method), this method will do nothing. This is because the
+  /// initialLifecycleState may already be stale and it no longer makes sense
```

Comment:
Should probably be `[initialLifecycleState]` if it is code, or separate words if it is a concept.